.github/workflows/go.yml

```diff
@@ -0,0 +1,43 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    env:
+      GO111MODULE: on   
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+
+    - name: Checkout SRC
+      run: |
+        go env
+        pwd
+        env
+        cd /home/runner/go
+        mkdir src/k8s.io -p
+        cd /home/runner/go/src/k8s.io
+        echo "Current Working Directory Below"
+        pwd
+        git clone https://github.com/kubernetes/kube-openapi.git
+
+    - name: Build
+      run: |
+        cd /home/runner/go/src/k8s.io/kube-openapi
+        go mod tidy
```

Comment:
Do we have the ability to run a diff to see if mod tidy changed anything, like in https://github.com/kubernetes/kube-openapi/blob/master/.travis.yml#L12

Reply:
added git diff just like the travis ci and rebased. thnx